_posts/2020-07-09-Steering-Group-executive-summary,-action-items,-and-minutes.md

```diff
@@ -24,12 +24,12 @@ We were joined by Franco Pestilli to discuss his potential BIDS grant. We went o
 ### Minutes
 
 - Discussed Franco Pestilli's grant
```

Comment:
FWIW: if already awarded, having a url to the grant (to at least get an overall clue on what it is about) would have been useful to outsiders.

Reply:
I thought the same but just assumed that it's just not "a public thing" right now.

cc @franklin-feingold

Reply:
he is currently working on it